packages/logger/tests/unit/configFromEnv.test.ts

```diff
@@ -169,6 +182,21 @@ describe('Class: EnvironmentVariablesService', () => {
     expect(value).toEqual(0.01);
   });
 
+  it.each([undefined, ''])(
```

Comment:
Since we removed all usage of `EnvironmentVariablesService` from the `Logger` class, we should also remove the files for the `EnvironmentVariablesService` itself - this should also make this test redundant.